src/components/meetings/docx/CouncilMeetingDocx.tsx

```diff
@@ -148,7 +150,7 @@ export const renderDocx = async ({ meeting, transcript, people, parties, speaker
     city: City,
     meeting: CouncilMeeting,
     transcript: Transcript,
-    people: Person[],
+    people: (PersonWithRelations | any)[],
```

Comment:
`any` shouldn't be an option. Removing it here will also remove the need for declaring it on the line above 
```suggestion
    people: (PersonWithRelations)[],
```